packages/core/src/crypto/jose/jwk/Ed25519Jwk.ts

```diff
@@ -0,0 +1,87 @@
+import type { JwkJson } from './Jwk'
+import type { Buffer } from '../../../utils'
+
+import { TypedArrayEncoder } from '../../../utils'
+import { KeyType } from '../../KeyType'
+import { JwaCurve, JwaKeyType } from '../jwa'
+import { JwaSignatureAlgorithm } from '../jwa/alg'
+
+import { Jwk } from './Jwk'
+import { hasKty, hasCrv, hasX, hasValidUse } from './validate'
+
+export class Ed25519Jwk extends Jwk {
```

Comment:
Not too sure how, but it might be good to note somewhere that we only use public values inside the JWK. 

Some JWK libraries have a `toPublicMaterial` function which strips all the private values, like `d` here for instance. 

Just some docs would suffice I think.

Reply:
I think my preference would be to include a `includePrivateKey: boolean` or somethign like that which by default has the value false.

However I'd like to not add it now, as we don't have modelled the private properties yet and then I'd need to implement that for all key types, although we never word with private keys in AFJ (at least not on this layer)